privaterelay/context_processors.py

```diff
@@ -12,13 +12,17 @@ def common(request):
     fxa = _get_fxa(request)
     avatar = fxa.extra_data['avatar'] if fxa else None
     accept_language = request.headers.get('Accept-Language', 'en-US')
-    country_code = request.headers.get('X-Client-Region', 'us').lower()
+    country_code = request.headers.get('X-Client-Region', '').lower()
```

Comment:
I'd do one of two things here:

1. Fall back to `accept_language` if this header is not available (risking that we're showing the wrong price/currency to people, and/or showing Premium promotion to people who can't get it), or
2. Set `country_code` to `us` if the header is not set and `DEBUG==True`, otherwise we'll never see Premium banners when running locally, possibly breaking it without noticing.

Reply:
Moved a lot of this into `utils.py` now too.

Reply:
Looks good. I guess the second point isn't too relevant if you fall back to the Accept language too, since the browser I think always sends an accept language and thus we'll never hit that code path in practice, but it doesn't do any harm either :)